src/pages/api/add-component.js

```diff
@@ -3,10 +3,10 @@ import shell from 'shelljs'
 
 export default async function handler(req, res) {
   try {
-    const response = await shellExec('bit --version')
-    const response2 = shell.exec('bit --version')
+    // const response = await shellExec('bit --version')
+    const response2 = shell.exec(`bit create tiui-react-oc ${req.body.path}`)
```

Comment:
This is potentially dangerous Validate and Sanitize before passing to exec

Reply:
Added: https://github.com/trilogy-group/tu2k22-openchakra/pull/32/commits/26d614ce1206e8782af2c67329728729610d0285